docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3610,10 +3619,11 @@ An `ActionProfileGroup` entity update message has the following fields:
       existing port on the device, otherwise the server must return
       `INVALID_ARGUMENT`.
 
-* `max_size` is the maximum sum of all member weights for the group. This field
-  is defined when the group is inserted, and must not be changed in a
-  `MODIFY` update. See the subsection below for the [rules on setting
-  `max_size`](#sec-max-size-rules).
+* `max_size` is the maximum sum of all members or member weights (as per the
+  `selector_size_semantics`) for the group. This field is defined when the group
+  is inserted, and must not be changed in a  `MODIFY` update, otherwise an
```

Comment:
nit: I spotted another double whitespace ...

Reply:
Nooooo! Mistakes were made. I even searched to make sure this was the last spot.